### DISCUSSION TITLE
How handle imported transfert operation

### DISCUSSION THREAD

**Discussion by Aohzan** [firefly-iii/firefly-iii#6814] [Firefly III] [answered]
Hello,
I have two accounts where I made a transfert between them
When the transaction is imported in ff, I have a transaction in the first account (debit) and an other in the second account (credit).
So I created a rule which convert the transaction of the first account into a transfert type, but now, I have two transaction in the second account, the transfert and the credit.

How can I handle this ?

Thank you

**Comment by JC5** [comment, marked as answer]
This is difficult to handle with rules, although there is an action to delete transactions. I would delete them manually if possible.

**Reply by Aohzan** [reply]
so I have to make a 2nd rule to delete the second transaction, because I have a lot of internal transfer
maybe it can be an idea for an improvement, for a rule with transfer transaction, we can choose to delete a matching transaction in the destination account ?
